1. Basic Web Development/server/src/Util/PermTypes.ts

```diff
@@ -0,0 +1,5 @@
+export enum PermTypes {
+    Nss = "permissionNss",
+    Pms = "permissionPms",
+    All = "permissionAll"
```

Comment:
nitpick: enum members should be easy to read and have suggestive names: `PermTypes.Nss` is maybe not as obvious as `PermissionType.Necessary` would be.